docs/main/user-guides/migration/migration-using-pv-migrate.md

```diff
@@ -0,0 +1,303 @@
+---
+id: migration-using-pv-migrate
+title: Migration from Legacy Storage to Latest Storage Solution
+keywords:
+ - Migration
+ - Data Migration
+ - Migration from OpenEBS Local PV Device to OpenEBS LVM Local PV
+ - Local PV Device to Local PV LVM
+ - Local PV Device to Local PV ZFS
+ - Migration from OpenEBS cStor to OpenEBS Replicated
+ - cStor to Replicated
+ - cStor to Mayastor
+ - Jiva to Replicated
+ - Jiva to Mayastor
+description: This section outlines the process of migrating the legacy storage to latest storage solution.
+---
+
+# Migration using pv-migrate
+
+This section describes the process of migrating the legacy storage to latest storage solution.
+
+## Overview
+
+OpenEBS data migration is the process of transferring data from the legacy storage to the latest storage solution. The data can be migrated from any non-OpenEBS storage to OpenEBS storage and this section provides detailed description about the migration process for legacy storage users.
```

Comment:
The above section needs correction, it's not communicating the right message.
Let me try phrasing it, you can build on that.

Data migration is the process of moving data from a source storage to a destination storage. In the OpenEBS context, users can migrate data from their legacy OpenEBS storage to the latest OpenEBS.
There are different techniques for performing data migration. One can perform data migration within the same Kubernetes cluster or across Kubernetes clusters. The below guides describe couple of approaches for migration from legacy OpenEBS to latest OpenEBS. Users of non-OpenEBS storage solutions can also use the approaches described below to migrate their data to OpenEBS storage.
1. Data migration using pv-migrate tool
2. Data migration using Velero tool